Assets/Scripts/Framework/Animation/AnimationController.cs

```diff
@@ -0,0 +1,39 @@
+using System.Linq;
+using UnityEngine;
+
+namespace Framework.Animation
+{
+    public sealed class AnimationController : MonoBehaviour
+    {
+        private Animator _animator;
+
+        private const string INVALID_ANIMATION = "Animation name does not exist in the Animator controller: ";
+        private const string NO_ANIMATOR_ERROR = "Animator component is not assigned.";
+
+        private void Awake() => _animator = GetComponent<Animator>();
+
+        public void PlayAnimation(string animationName)
+        {
+            if (IsValidAnimation(animationName))
+                _animator.SetTrigger(animationName);
+        }
+    
+        private bool IsValidAnimation(string animationName)
+        {
+            if (_animator == null)
+            {
+                Debug.LogError(NO_ANIMATOR_ERROR);
+                return false;
+            }
+
+            bool animationParameters = _animator.parameters.Any(animationParam  => animationParam .name == animationName);
+            if (!animationParameters)
```

Comment:
```suggestion
            bool animationParameters = _animator.parameters.Any(animationParam  => animationParam .name == animationName);
            
            if (!animationParameters)
```
[Line er tussen.](https://github.com/Team-Swamp/IceBites/wiki/Code-convention#if-statement-and-loop)